### PR TITLE
Remove Implementation section from issue templates

### LINKED ISSUE #989: Remove "Implementation" section from issue templates
## Suggested Improvement

This is a follow-up to https://github.com/WordPress/openverse/issues/329 in our primary repo.

We have a few other issue templates, namely `.github/ISSUE_TEMPLATE/infrastructure_improvement.md` and `.github/ISSUE_TEMPLATE/code_quality.md` which also have "Implementation" sections which can be removed.

### PR DESCRIPTION
**Pull request by miikkuu**
## Fixes

Fixes #989 #991

## Description

removed extra implementations and extra lines .

### CODE REVIEW

**Comment by miikkuu** [comment]
@dhruvkb

File: .github/ISSUE_TEMPLATE/code_quality.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 name: Code Quality Improvement Suggestion
 about: Suggest a change that does not add a feature
 labels: "🚦 status: awaiting triage"
-title: "<Replace this with the actual title>"
```
Comment (dhruvkb) [inline .github/ISSUE_TEMPLATE/code_quality.md:5]: Please don't remove the title, that is a required field.

File: .github/ISSUE_TEMPLATE/code_quality.md [hunk 2]
```diff
@@ -21,8 +20,3 @@ title: "<Replace this with the actual title>"
 
 <!-- Add any other context suggestion here. -->
```
Comment (dhruvkb) [inline .github/ISSUE_TEMPLATE/code_quality.md:22]: Please remove this blank line. That is causing the [CI to fail](https://github.com/WordPress/openverse-catalog/actions/runs/4110774618/jobs/7093917647#step:7:52).

You can also run lint locally to ensure that this file is cleaned up before pushing. For that you will need to run these two commands:

```bash
$ just install
$ just lint
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
I've marked changes in one file but they apply to both files identically.

File: .github/ISSUE_TEMPLATE/code_quality.md (hunk 1 above)
Comment (miikkuu) [reply .github/ISSUE_TEMPLATE/code_quality.md:5]: so sir do you mean it should show empty content , like:
title:
?

Comment (dhruvkb) [reply .github/ISSUE_TEMPLATE/code_quality.md:5]: No, I meant to leave the line unchanged. The way it was before.

**Review by AetherUnbound — COMMENTED** [review]
+1 to the changes requested by Dhruv 🙂

**Review by AetherUnbound — APPROVED** [review]

**Pull request merged**